tests/mpi/broadcast_01.cc

```diff
@@ -68,6 +70,59 @@ check(const std::vector<int> &int_vector, const std::string &string)
   deallog << std::endl;
 }
 
+void
+check(int int_value)
+{
+  const auto my_proc = Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
+
+  int buffer;
+  if (my_proc == 0)
+    {
+      buffer = int_value;
+    }
+
+  const auto result = Utilities::MPI::broadcast(MPI_COMM_WORLD, buffer);
```

Comment:
I don't think this actually works. Don't you need to provide the root process somewhere? Same for the other calls below.

Reply:
See https://github.com/dealii/dealii/blob/25c174d96bfa1fa8e25b256a0a3aa9bba124b72a/include/deal.II/base/mpi.h#L1155-L1159. The root process is defaulted to be 0.